src/README.md

```diff
@@ -19,15 +20,28 @@
 |                          | `ui/stats`             |                 |
 |                          | `ui/chat`              |                 |
 |                          | `targets/target`       |                 |
-  
-  
+
 ### Main file descriptions
-| File           | Description                                                                                                                                    |
-|----------------|------------------------------------------------------------------------------------------------------------------------------------------------|
-| `main.js`      | Sets up phaser and loads all of the scenes. Scene load order affects the z-index                                                               |
-| `load.js`      | Init cookies, load item/panel images, and dynamically import all item classes from `*-manifest.js` files and store them in a dictionary        |
-| `cookie-io.js` | Contains getter/setters for cookie obj. The obj reference is passed between all scenes to get/set save data                                    |
-| `main-menu.js` | Starts music from `scenes/audio.js`, lets you delete cookies, and start on the last level the user was on                                      |
-| `level.js`     | Parent class for all levels. Creates the background, targets, and dashboard UI. Also checks quest completion and animates weapon swing         |
-| `dashboard.js` | Contains the visuals and objects for the right portion of the screen (inventory, skills, etc.)                                                 |
 
+| File           | Description                                                                                                                             |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| `main.js`      | Sets up phaser and loads all of the scenes. Scene load order affects the z-index                                                        |
+| `load.js`      | Init cookies, load item/panel images, and dynamically import all item classes from `*-manifest.js` files and store them in a dictionary |
+| `cookie-io.js` | Contains getter/setters for cookie obj. The obj reference is passed between all scenes to get/set save data                             |
+| `main-menu.js` | Starts music from `scenes/audio.js`, lets you delete cookies, and start on the last level the user was on                               |
+| `level.js`     | Parent class for all levels. Creates the background, targets, and dashboard UI. Also checks quest completion and animates weapon swing  |
+| `dashboard.js` | Contains the visuals and objects for the right portion of the screen (inventory, skills, etc.)                                          |
+
+## Where to add content
```

Comment:
I think whenever you need to modify multiple areas of code to do something it should be documented. I get tilted at work when I make a change but it turns out there's some other part that I need to update that's not obvious

Reply:
Ehh I won't make you remove it but I don't like this section. If you are adding stuff you should work through the logic and understand the full flows. Having a cheat sheet like this seems lazy and annoying to maintain. Depending on what needs to be added, it may not even be relevant or it may sway a developer to force his code into the prescribed buckets instead of taking a better solution.

Reply:
I think it's pretty trivial to update and in cases where you need to update files that are not in the same area they can easily be overlooked